lib/Dialect/Calyx/Export/CalyxEmitter.cpp

```diff
@@ -545,8 +541,12 @@ void Emitter::emitComponent(ComponentOp op) {
                 SubLibOp, ShruLibOp, RshLibOp, SrshLibOp, LshLibOp, AndLibOp,
                 NotLibOp, OrLibOp, XorLibOp, WireLibOp>(
               [&](auto op) { emitLibraryPrimTypedByFirstInputPort(op); })
-          .Case<MultPipeLibOp, DivUPipeLibOp, DivSPipeLibOp>(
+          .Case<MultPipeLibOp>(
               [&](auto op) { emitLibraryPrimTypedByFirstOutputPort(op); })
+          .Case<RemUPipeLibOp, RemSPipeLibOp, DivUPipeLibOp, DivSPipeLibOp>(
+              [&](auto op) {
+                emitLibraryPrimTypedByFirstOutputPort(op, {"std_div_pipe"});
```

Comment:
Also perhaps `"std_div_pipe"` (no brackets) isn't sufficient here? Also perhaps `/*calyxLibName=*/`

Reply:
`const char*` to `Optional<StringRef>` seems to a be one step too much for the compiler to deduce, hence the brackets.